charts/weave-gitops/templates/wego-app.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wego-app
+  namespace: {{.Values.namespace}}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wego-app
+    spec:
+      serviceAccountName: wego-app-service-account
+      containers:
+        - name: wego-app
+          image: "ghcr.io/weaveworks/wego-app:{{.Chart.AppVersion}}"
+          args: ["ui", "run", "-l", "--helm-repo-namespace","{{.Values.namespace}}"]
```

Comment:
This might change with @SamLR's work.

Reply:
👍  the `ui` command is disabled now

Reply:
(sorry missed this)

The `run` command is also not needed.

We should also discuss whether the image will still be called `wego-app` as the binaries are currently called `gitops` and `gitops-server`. @JamWils any preferences?

We should also make sure we generate the image(s) and I guess we may want to consider adding a place holder one so that this helm chart will actually work.